packages/admin/cms-admin/src/generator/generatePage.ts

```diff
@@ -0,0 +1,43 @@
+import { IntrospectionQuery } from "graphql";
+
+import { CrudGeneratorConfig } from "./types";
+import { buildNameVariants } from "./utils/buildNameVariants";
+import { camelCaseToHumanReadable } from "./utils/camelCaseToHumanReadable";
+import { writeGenerated } from "./utils/writeGenerated";
+
+export async function writeCrudPage({ entityName, target: targetDirectory }: CrudGeneratorConfig, schema: IntrospectionQuery): Promise<void> {
```

Comment:
This might not be async as no Promise is returned (maybe writeGenerated needs to be awaited or returned)

Reply:
there was an await missing....